popup.css

```diff
@@ -0,0 +1,213 @@
+h1,
+p {
+  padding: 0;
+  margin: 0;
+}
+
+.section {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  background-color: rgb(0, 0, 0, 90%);
+  display: flex;
+}
+
+.popup__section {
+  display: flex;
+  gap: 20px;
+  position: relative;
+  margin: 24px;
+  width: 95%;
+  height: 90vh;
+  background-color: white;
+  flex-direction: column;
+  padding: 16px;
+}
+
```

Comment:
- [ ] It is required to stick to the guiding design as close as possible. In your popup window, there's more than required space between project image and the description. Kindly fix that.
- [ ] The width of popup box is wider than what we have on the Figma template. Kindly reduce the width to match the figma design.
- [ ] Also, there's more than required left-margin at the left side of the project image. Kindly look into this and fix.

### Your Design
![your-popup-desktop](https://user-images.githubusercontent.com/26597432/216331751-8533147f-d150-49e5-8d10-60d5e512f456.png)

### Figma Design
![temp-popup](https://user-images.githubusercontent.com/26597432/216331796-9e0a12a2-5e86-43d4-87a1-cadb1a677356.png)

 

- [ ] For the mobile version, the whole information on the popup card is not visible and while trying to scroll up down to see the rest of the information, there is overlay information and the rest of the information cannot be seen. Please, take a look.
[your-popup-mobile.webm](https://user-images.githubusercontent.com/26597432/216333821-fcf6be45-a870-4d8d-8503-9b4e3f4d03f8.webm)